src/clients/Fabric.js

```diff
@@ -1088,6 +1088,35 @@ const Fabric = {
 
   /* Object creation / modification */
 
+  GetContentAdminsGroupAddress: async () => {
+    try {
+      // Attempt to add newly created object to content admins group
```

Comment:
This comment doesn't apply here - right?  We're just retrieving the address of the content admins group

Reply:
yeah, I moved some stuff around after I wrote this, originally contained the actual permission grant